webapp/src/App.tsx

```diff
@@ -8,6 +8,8 @@ import Recipe from './pages/Recipe';
 import RecipeList from './pages/RecipeList';
 import UserProfile from './pages/UserProfile/UserProfile';
 import ButtonArsh from './components/atoms/ButtonArsh';
+import UsersQuery from './components/atoms/UsersQuery';
+import { createClient, Provider } from 'urql';
```

Comment:
Did you want to use the urql provider or the react-graphql-ui provider? I think either is fine here but you may want to change it going forward

Reply:
tbh since we're getting close to friday whichever is faster just wanted you to think about it!